modules/aws/ecr_test.go

```diff
@@ -0,0 +1,23 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestEcrRepo(t *testing.T) {
+	t.Parallel()
+
+	region := GetRandomStableRegion(t, nil, nil)
+	repo1, err := CreateECRRepoE(t, region, "terratest")
+	defer DeleteECRRepo(t, region, repo1)
+
+	assert.Nil(t, err)
```

Comment:
```suggestion
	require.NoError(t, err)
```

So we don't do the rest of the steps and panic if there is an error when we dereference the pointer in the next step when there is an error.

This applies to the similar checks done below.

Reply:
Done in 2840220